js/src/views/ParityBar/parityBar.css

```diff
@@ -55,6 +55,7 @@
   overflow: auto;
   display: flex;
   background: rgba(0, 0, 0, 0.8);
+  min-height: 300px;
```

Comment:
This should be `min-height = .expanded.max-height (300) - (.header.height + 2*.header.padding)`, thus around `248px`